docs/source/getting-started/testing.rst

```diff
@@ -0,0 +1,134 @@
+.. Copyright (c) 2020 The DAML Authors. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Testing Your App
+****************
+
+When developing your application, you will want to test certain workflows through the UI.
+In particular, you'll want to make sure that changes such as new features do not break existing functionality by mistake.
+It may be fine to do this testing manually at first, but it quickly becomes more tiresome and error prone as the app grows.
```

Comment:
I don't think the last two sentences are necessary here. We don't want to tell people _why_ they should test but rather _how_ to do it.